Libraries/Components/WebView/WebView.android.js

```diff
@@ -127,13 +134,25 @@ var WebView = React.createClass({
       domStorageEnabled = this.props.domStorageEnabledAndroid;
     }
 
+    var htmlProp, urlProp, sourceProp;
+    if(this.props.html) {
+      console.warn('html is deprecated. Use source instead.');
+      htmlProp = this.props.html;
+    } else if(this.props.url) {
+      console.warn('url is deprecated. Use source instead.');
+      urlProp = this.props.url;
+    } else {
+      sourceProp = this.props.source;
+    }
```

Comment:
In the interests of code cleanliness, you can probably just remove the native html and url props completely, and implement support for them on the JS side by using them to build the source object. Something like:

    var source = this.props.source || {};
    if (this.props.html) {
      console.warn('html is deprecated. Use source instead.');
      source.html = this.props.html;
    } else if (this.props.url) {
      console.warn('url is deprecated. Use source instead.');
      source.url = this.props.url;
    }

    <RCTWebView
        ref={RCT_WEBVIEW_REF}
        key="webViewKey"
        style={webViewStyles}
        source={source} // no need for url/html any more
        ...
    />